doc/data/messages/c/cell-var-from-loop/good/new_function.py

```diff
@@ -0,0 +1,13 @@
+def teacher_greeting(names):
+    def greet(name):
+        # do something
+        print(f"Hello, {name}!")
+
+    for name in names:
+        if name.isalpha():
+            # we're passing the value of "name" to the function here
+            greet(name)
+
+
+teacher_greeting(["Alice", "Bob", "Charlie", "Not-A-Name"])
+# "Hello, Alice!", ...
```

Comment:
```suggestion
teacher_greeting(["Graham", "John", "Terry", "Eric", "Terry", "Michael"])
# "Hello, Graham"
# "Hello, John"
# "Hello, Eric"
# "Hello, Terry"
# "Hello, Michael"
```
This is a Monthy Python reference, let's keep it :p

Reply:
Aha, of course!